src/gleam/map.gleam

```diff
@@ -19,7 +19,10 @@ if javascript {
 /// See [the Erlang map module](https://erlang.org/doc/man/maps.html) for more
 /// information.
 ///
-pub external type Map(key, value)
+pub external type Map(
+  key,
+  value,
+)
```

Comment:
Huh, why is this split onto multiple lines? Seems like a bug

Reply:
The github ci/cd runs against gleam nightly (I think).
I had to change to gleam nightly and run `gleam format` to make the CI/CD pass.